### ISSUE TITLE
I am the author of CryptoMiniSat, I'd like to get control of pycryptosat

### ISSUE THREAD

**Issue by msoos** [pypa/warehouse#10060]
Hi,

I am Mate Soos, the author of [CryptoMiniSat](https://github.com/msoos/cryptominisat), as per my GitHub profile, and as per how the original PyPi maintainer put together the [pycryptosat PyPi project](https://pypi.org/project/pycryptosat/). Check out the homepage (www.msoos.org), the Maintainer (clearly set to me), etc. The "pycryptosat" is the python module that CryptoMiniSat builds&installs when built using `cmake` and python2/3 is present.

The issue is, the version on PyPi very outdated. I tried reaching out the maintainer, but I couldn't get any response. There is no official email address listed for her/him, so I tried searching around. I sent a few emails to what I would guess to be her/his email, but no response. I'd like to get control over the PyPi project. It's currently very outdated, and I believe I would be in a good position to maintain it. Furthermore, I'd like to release some research projects in the long term (I work in academia sometimes), and having control over pycryptosat would be invaluable for this.

Basically, the best would be if you have access to her/his email address, could you please write an email to them? I have a feeling they'd be happy to give the project to me, to be honest -- there is no reason to believe they wouldn't trust me :) If they also don't respond to you, would it be possible to give me the project after some waiting & discussion?

Thanks so much in advance,

Mate

**Comment by yeraydiazdiaz** [comment]
Hi @msoos, please open a [PEP 541 request](https://github.com/pypa/pypi-support/issues/new?assignees=&labels=PEP+541&template=pep541-request.yml&title=PEP+541+Request%3A+PROJECT_NAME) for your project.

**Comment by msoos** [comment]
Thanks a lot @yeraydiazdiaz and sorry for not knowing!

Mate

**Comment by yeraydiazdiaz** [comment]
No problem, it's not like "PEP 541" is a super descriptive name 😄
